deploy/crds/onepassword.com_onepassworditems_crd.yaml

```diff
@@ -38,8 +36,43 @@ spec:
             type: object
           status:
             description: OnePasswordItemStatus defines the observed state of OnePasswordItem
+            properties:
+              conditions:
+                description: 'Important: Run "operator-sdk generate k8s" to regenerate
+                  code after modifying this file Add custom validation using kubebuilder
+                  tags: https://book-v1.book.kubebuilder.io/beyond_basics/generating_crd.html'
+                items:
+                  properties:
+                    lastTransitionTime:
+                      description: Last time the condition transit from one status
+                        to another.
+                      format: date-time
+                      type: string
+                    message:
+                      description: Human-readable message indicating details about
+                        last transition.
+                      type: string
+                    status:
+                      description: Status of the condition, one of True, False, Unknown.
+                      type: string
+                    type:
+                      description: Type of job condition, Completed.
+                      type: string
+                  required:
+                  - status
+                  - type
+                  type: object
+                type: array
+              ready:
+                description: True when the Kubernetes secret is ready for use.
+                type: boolean
+            required:
+            - conditions
             type: object
           type:
-            description: 'Kubernetes secret type. More info: https://kubernetes.io/docs/concepts/configuration/secret/#secret-types'
```

Comment:
To not lose the description of this field of the `OnePasswordItem`, add the following comment above the field's definition in `pkg/apis/onepassword/v1/onepasswordsecret_types.go` (lines 49-56)

```go
// Kubernetes secret type. More info: https://kubernetes.io/docs/concepts/configuration/secret/#secret-types
```

Reply:
Good catch. Fixed.